src/main/java/org/reactome/release/qa/check/HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities.java

```diff
@@ -19,18 +16,19 @@
 @SliceQATest
 public class HumanReactionsWithoutDiseaseAndHaveNonHumanPhysicalEntities extends AbstractQACheck {
 
-    private static final String innateImmunityPathwayDbId = "168249";
+    private static final String INNATE_IMMUNITY_PATHWAY_DBID = "168249";
+    private static List<String> skiplistDbIds = new ArrayList<>();
 
     @Override
     public QAReport executeQACheck() throws Exception {
         QAReport report = new QAReport();
-        QACheckerHelper.setSkipList(Arrays.asList(innateImmunityPathwayDbId)); // Innate Immunity Pathway
         QACheckerHelper.setHumanSpeciesInst(dba);
+        skiplistDbIds.add(INNATE_IMMUNITY_PATHWAY_DBID);
```

Comment:
You could make this `this.skiplistDbIds` so it is clear it is an instance variable.